src/types/shortcuts.ts

```diff
@@ -31,12 +31,22 @@ export const tokenAmountWithMetadata = z.object({
   isNative: z.boolean(),
 })
 
+export const ZodEnableSwapFee = z
```

Comment:
this is a post request, can we just use zod booleans?

Reply:
I don't think so:

> Note — Boolean coercion with z.coerce.boolean() may not work how you expect. Any [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) value is coerced to true, and any [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) value is coerced to false.

Reply:
doh, yes we can.